app/app/urls.py

```diff
@@ -156,6 +156,16 @@
         dashboard.views.verify_user_twitter,
         name='verify_user_twitter'
     ),
+    url(
+        r'^api/v0.1/profile/(?P<handle>.*)/request_verify_google',
+        dashboard.views.request_verify_google,
+        name='request_verify_google'
+    ),
+    url(
+        r'^api/v0.1/profile/verify_user_google',
+        dashboard.views.verify_user_google,
+        name='verify_user_google'
+    ),
```

Comment:
^ not very keen on having 2 new endpoints
Ideally I'd say we have one common endpoint 

handle/verify/ as a POST operation and based on a body param -> we could decide which flow to send it into 

cc @apbendi

Reply:
Seems good, I will going for that.

Reply:
I realized google callback URL can not be dynamic, so we can not include the handle parameter in the callback URL. 
Another issue, google uses the GET method to call callback URL.
But existing twitter callback uses POST and getting a handle parameter in the URL. to merge them I need to change that.
@thelostone-mc what do you think?

Reply:
I think it's fine to have separate endpoints